### ISSUE TITLE
"Age at diagnosis" chart: range should not overlap

### ISSUE THREAD

**Issue by aalex** [kids-first/kf-portal-ui#1835]
The columns of the Age at diagnosis chart overlap in their range.

## How to reproduce
- In the cohort builder, search by "Diagnosis (Mondo)" with value "Chordoma".

![image](https://user-images.githubusercontent.com/293677/59446914-6a70c480-8dd0-11e9-8ed7-20e174fbf29e.png)

- There should be 4 participants in the result set. (currently)
- Click on the "Summary View" tab. 
- The "Age at diagnosis" chart shows columns that overlap. (one ends at 5, and the next starts at 5, which is the same value)  

Instead, there should be a way to know which column ends or start exactly at a given number. Let's use a range that is `[0, 5[`, `[5, 10[`, `[10,15[`. That means that what we will show to the user is `0-4`, `5-9`, `10-14`. (in this notation, the `N]` means "up to N, including N", whereas `N[` means "up to N, exluding it" - up to N-1.

## Acceptance criteria
In short:
- add square brackets around the numbers, for each column. Example: `[0 - 5[` for the first column, and `[5 - 10[` for the second, and so on. That means that the first columns has items whose value is between 0 and 4.99999, and then the second one has items whose value is between 5 and 9.9999.

## Other observations
It shows results as if there were 5 participants in the data set, but there are 4. Instead, one might think that the total number of participants in that chart should be 4, but it is so because  some participants have more than one diagnosis.

## Screenshot
![image](https://user-images.githubusercontent.com/293677/59446520-ad7e6800-8dcf-11e9-8f43-365af82a38a1.png)

## See also
- #1681
- #1454

**Comment by aalex** [comment]
Two observations: 

1) It's not clear whether each range includes the number or not. (for example, 5-10: does it include 5 or not?)

2) It seems like graph should filter out diagnosis related to the search query. (and that is what is happening here)

**Comment by aalex** [comment]
TODO: make sure the click on the columns still work - because what is used right now it the contents of the label.

**Comment by aalex** [comment]
TODO updat ethis issue - only the square brackets.

**Comment by aalex** [comment]
The changes requested are detailed in another ticket: #1903
